pkg/providers/vmware/client/client.go

```diff
@@ -0,0 +1,86 @@
+package client
+
+import (
+	"context"
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/vim25/types"
+	"net/url"
+)
+
+// RichOvirtClient is responsible for retrieving VM data from oVirt API
+type richVmwareClient struct {
+	client *govmomi.Client
```

Comment:
Why not to add `context` to this struct so it can be used for all interactions with the govmomi client?